src/test/unit/BLSRegistryCoordinatorWithIndicesUnit.t.sol

```diff
@@ -665,6 +682,54 @@ contract BLSRegistryCoordinatorWithIndicesUnit is MockAVSDeployer {
         registryCoordinator.registerOperatorWithCoordinator(quorumNumbers, operatorToRegisterPubKey, defaultSocket, operatorKickParams, signatureWithSaltAndExpiry);
     }
 
+    function testEjectOperatorFromCoordinator_Valid() public {
```

Comment:
might be nice to have a version of this that only ejects the operator from some quorums?
also would really appreciate some comments on these tests!!

Reply:
c4f9e92a done :)